src/xcode/ENA/ENA/Source/Scenes/Home/HomeInteractor.swift

```diff
@@ -161,9 +162,11 @@ final class HomeInteractor {
 	}
 
 	private func initialCellConfigurators() -> [(section: HomeViewController.Section, cellConfigurators: [CollectionViewCellConfiguratorAny])] {
+
 		let currentState = stateHandler.getState()
 		activeConfigurator = HomeActivateCellConfigurator(state: currentState)
-		let dateLastExposureDetection = store.dateLastExposureDetection
+
+		let dateLastExposureDetection = Date(timeIntervalSinceNow: -232000) //store.dateLastExposureDetection
```

Comment:
why is this commented out? + maybe pull out the magic constant

Reply:
it was testing code, thanks for catching it